src/platform/plugins/shared/saved_search/server/saved_objects/search.ts

```diff
@@ -81,6 +83,18 @@ export function getSavedSearchObjectType(
           create: SCHEMA_SEARCH_MODEL_VERSION_5,
         },
       },
+      6: {
+        changes: [
+          {
+            type: 'unsafe_transform',
+            transformFn: (typeSafeGuard) => typeSafeGuard(extractTabsTransformFn),
+          },
+        ],
+        schemas: {
+          forwardCompatibility: SCHEMA_SEARCH_MODEL_VERSION_6.extends({}, { unknowns: 'ignore' }),
+          create: SCHEMA_SEARCH_MODEL_VERSION_6,
+        },
+      },
```

Comment:
This looks right to me, but we'll definitely want to get a good look at it from @elastic/kibana-core to make sure it's done correctly.